### ISSUE TITLE
Kebab | Meatball | Hamburger Menues (4/30)

### ISSUE THREAD

**Issue by roblevintennis** [AgnosticUI/agnosticui#83]
This is blocked by initial steps of:

1. Build [CSS package](https://github.com/AgnosticUI/agnosticui/issues/174) component ✅
2. Build [first Vue component](https://github.com/AgnosticUI/agnosticui/issues/180) ✅
3. [x] [React & React Examples](https://github.com/AgnosticUI/agnosticui/issues/183) ✅
4. [x] [Svelte & Svelte Examples](https://github.com/AgnosticUI/agnosticui/issues/175) ✅
- [x] Update Docs & Astro Playgrounds
- [x] Publish frameworks

### Nice to have -- Angular

This has its own ticket and will be done later
- [ ] [Angular & Angular Examples](https://github.com/AgnosticUI/agnosticui/issues/184)
